### ISSUE TITLE
[onert] Android build fail on ubuntu 20.04

### ISSUE THREAD

**Issue by hseok-oh** [Samsung/ONE#10052]
I found out onert android build fail on ubuntu 20.04

```
In file included from /home/nfs/git/ONE/compute/ARMComputeEx/src/core/CL/kernels/CLArgMinMaxLayerKernelEx.cpp:40:
/home/nfs/git/ONE/compute/ARMComputeEx/arm_compute/core/CL/kernels/CLArgMinMaxLayerKernelEx.h:43:10: fatal error: 'src/core/CL/ICLKernel.h' file not found
#include "src/core/CL/ICLKernel.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~
make[3]: Entering directory '/home/nfs/git/ONE/Product/aarch64-android.debug/obj'
[  6%] Building CXX object compute/ARMComputeEx/CMakeFiles/arm_compute_ex.dir/src/core/CL/kernels/CLBinaryLogicalOpKernel.cpp.o
[  6%] Built target nnapi_gtest
[  6%] Built target absl_bad_any_cast_impl
Consolidate compiler generated dependencies of target test_cker
make[3]: Leaving directory '/home/nfs/git/ONE/Product/aarch64-android.debug/obj'
make[3]: Entering directory '/home/nfs/git/ONE/Product/aarch64-android.debug/obj'
make[3]: Entering directory '/home/nfs/git/ONE/Product/aarch64-android.debug/obj'
1 error generated.
make[3]: *** [compute/ARMComputeEx/CMakeFiles/arm_compute_ex.dir/build.make:90: compute/ARMComputeEx/CMakeFiles/arm_compute_ex.dir/src/core/CL/kernels/CLArgMinMaxLayerKernelEx.cpp.o] Error 1
```

**Comment by hseok-oh** [comment]
Fail to find header in `src` directory. Interesting why it fails on android, ubuntu 20.04 only.

**Comment by hseok-oh** [comment]
Confirmed build fail is resolved by #10053.
